transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/hypercube/HighDimSpec.groovy

```diff
@@ -424,17 +434,18 @@ class HighDimSpec extends RESTSpec {
                 path: PATH_HIGH_DIM,
                 acceptType: acceptType,
                 query: [
-                        assay_constraint: new JsonBuilder([type: ConceptConstraint, path: "\\Public Studies\\EHR\\Vital Signs\\Heart Rate\\"])
+                        type: 'rnaseq_transcript',
+                        constraint: new JsonBuilder([type: ConceptConstraint, path: "\\Public Studies\\EHR\\Vital Signs\\Heart Rate\\"])
                 ],
-                statusCode: 500
+                statusCode: 400
         ]
 
         when:
         def responseData = get(request)
 
         then:
-        that responseData.httpStatus, is(500)
-        that responseData.type, is('IllegalStateException')
+        that responseData.httpStatus, is(400)
```

Comment:
I changed the server code for this. 5xx indicates a server side problem, 4xx indicates a problem with the request the client made. A 422 would be more appropriate than a 400 but the server side works by throwing an InvalidQueryException which grails maps to a 400 response. I didn't dive into how to change that.